Create CONTRIBUTING.md (#26)
COVID-science-engine/crowdsourced-papers-frontend: README.md

rails server
```

Then go to [http://localhost:3000](http://localhost:3000) to view the app.

> Current we have some basic authentication in front of the website. We will be replacing this with something better soon.

Reviewer: ```suggestion
Currently, we have some basic authentication in front of the website. We will be replacing this with something better soon.
```

> In development, the username and password is `foo` /  `bar`.

Reviewer: That's good to know, too. Had to look for it in the code to get it running. :P

```

# Contributing
We are looking for new contributors to help us build this project.

👉 Please read our [Contributing guide](CONTRIBUTING.md) to get started. 👈

> You can also join are [Discord server](https://discord.gg/V6kzVAS) to ask questions.

Reviewer: Little typo and I'd also formulate the end a bit more openly (instead of limiting it to asking questions).

```suggestion
You can also join our [Discord server](https://discord.gg/V6kzVAS) to get in touch.
```
